app/models/miq_server/worker_management/kubernetes.rb

```diff
@@ -12,10 +12,6 @@ def sync_from_system
     # we only have to sync the list of pods and deployments once
     ensure_kube_monitors_started if my_server_is_primary?
 
-    # Before syncing the workers check for any orphaned worker rows that don't have
-    # a current pod and delete them
-    cleanup_orphaned_worker_rows
-
     # Update worker deployments with updated settings such as cpu/memory limits
     sync_deployment_settings
```

Comment:
TODO I think this has to be after `cleanup_orphaned_worker_rows`, and doesn't make sense IMO to have `sync_deployment_settings` in `sync_from_system` since this is supposed to just be "find what is out there"